src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && (
+          <ZapMessageContainer onClick={openSearchList}>
+            ⚡ Or <HighlightText> select a token </HighlightText> in your wallet to ZAP
+          </ZapMessageContainer>
```

Comment:
Is this whole container meant to be clickable or just the text 'link'? 
![image](https://user-images.githubusercontent.com/18649057/169622182-33c6bd1f-9a41-4a7b-a60a-ebe701b446d2.png)

Reply:
either is fine from a ux perspective